Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/outbound/deferred/request/PassthroughOutboundPatientDiscoveryDeferredRequestTest.java

```diff
@@ -56,29 +58,29 @@ public void invoke() {
         AssertionType assertion = new AssertionType();
         NhinTargetCommunitiesType targets = new NhinTargetCommunitiesType();
         MCCIIN000002UV01 expectedResponse = new MCCIIN000002UV01();
-
```

Comment:
webContextProperties below here is not used.
Properties webContextProperties = new Properties();